modules/util/get_set_value.js

```diff
@@ -49,5 +43,12 @@ export function utilGetSetValue(selection, value, shouldUpdate) {
         shouldUpdate = (a, b) => a !== b;
     }
 
+    // only certain input element types allow manipulating the cursor
+    // see https://html.spec.whatwg.org/multipage/input.html#concept-input-apply
+    const supportedTypes = ['text', 'search', 'url', 'tel', 'password'];
+    if (!supportedTypes.includes(this.type)) {
```

Comment:
This does not quite work as intended, as `this` is here the global (`window`) object, and not the input element we'd want to check (d3's `selection.each' binds the `this` variable to the DOM element in the callback function).

So, in practice, this condition is always fulfilled, even for input element types which would support cursor manipulation. As it still fixes the textarea/raw tag editor bug, I think we can keep it for the moment (even though #9233 is now an issue again), but I'll see if I can fix this a little bit better. :blush: